payment-manager/src/main/java/com/qpay/paymentmanager/client/NotificationClientFeign.java

```diff
@@ -0,0 +1,16 @@
+package com.qpay.paymentmanager.client;
+
+import com.qpay.libs.models.PaymentNotification;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+
+@FeignClient(name = "notification", url = "${webclient.services.notification-manager.base-url}")
+public interface NotificationClientFeign {
```

Comment:
Feign is a library. You have to name it by its purpose. In your case just NotificationClient